src/test/test_script.py

```diff
@@ -200,8 +200,9 @@ def test_scriptpubkey_csv_2of2_then_1_from_bytes(self):
             (MPK_2, MPK_2_LEN, 1, 0, None, out_len), # Null output
         ]
         for args in invalid_args:
-            ret = wally_scriptpubkey_csv_2of2_then_1_from_bytes(*args)
-            self.assertEqual(ret, (WALLY_EINVAL, 0))
+            for fn in [wally_scriptpubkey_csv_2of2_then_1_from_bytes,
+                       wally_scriptpubkey_csv_2of2_then_1_from_bytes_opt]:
+                self.assertEqual(fn(*args), (WALLY_EINVAL, 0))
 
         # Valid cases
```

Comment:
Should we add also a valid test case? It may not be worth it though, it does not add much value.

Reply:
Yes, I will add valid cases once we decide if we will expose this variant.